packages/patternfly-4/react-core/src/declarations.d.ts

```diff
@@ -1,6 +1,6 @@
 // We use a JS babel plugin to resolve these imports that `tsc` doesn't
 // know about, so pretend that everything will work
-declare module '@patternfly/patternfly/components/*.css' {
+declare module '@patternfly/patternfly/*.css' {
```

Comment:
What is this fixing?  Can you add that to the description?

Reply:
It allows the import `@patternfly/patternfly/utilities/Accessibility/accessibility.css` to not fail typechecking.